js/site-search.js

```diff
@@ -157,19 +155,22 @@ bcpl.siteSearch = (($, window, constants) => {
 		.on('click', siteSearchTabSelector, onSearchTabClick)
 		.on('click', siteSearchSearchIconSelector, onSearchIconClick)
 		.on('click', searchButtonCatalogSelector, onSearchCatalogClick)
-		.on('keyup', siteSearchInputSelector, onSiteSearchKeyup)
 		.on('click', searchButtonEventsSelector, onSearchEventsClick)
-		.on('click', searchButtonWebsiteSelector, onSearchWebsiteClick);
+		.on('click', searchButtonWebsiteSelector, onSearchWebsiteClick)
+		.on('keyup', siteSearchInputSelector, onSiteSearchKeyup);
 
 	// Initially set up the catalog search
 	$(onSearchCatalogClick);
 
-	/* test-code */
+	// On Document Ready
+	$(() => {
+		clearCatalogSearch();
+	});
 
+	/* test-code */
```

Comment:
Test code?

Reply:
Yeah -- it allows us to expose methods just for testing. When you compile, Gulp strips it out.

Reply:
It's so you can test "private" methods without including them in "production".  A gulp task will rip this out when building the scripts.

It's not a perfect system, but works with for us and the revealing module pattern.